src/image-target/index.js

```diff
@@ -1,11 +1,13 @@
-const {Controller} = require('./controller');
+const {Controller, WARMUP_COUNT_TOLERANCE, MISS_COUNT_TOLERANCE} = require('./controller');
```

Comment:
I'd prefer not to expose these two constants from controller. Especially not expose to window.MINDAR.IMAGE. They are more like an internal constants. 

I think we could make these optional in the aframe component. So if they are not given, the default values from the controller will be picked up anyway.

Reply:
I removed the export of the constants. But it is hard to find out the default values by looking at the aframe.js file.